webapp/channels/src/components/setting_item_max.tsx

```diff
@@ -231,7 +224,12 @@ export default class SettingItemMax extends React.PureComponent<Props> {
                 className={`section-max form-horizontal ${this.props.containerStyle}`}
             >
                 {title}
-                <div className={widthClass}>
+                <div
+                    className={classNames('sectionContent', {
+                        'col-sm-12': this.props.isFullWidth,
+                        'col-sm-10 col-sm-offset-2': !this.props.isFullWidth,
+                    })}
```

Comment:
@abhijit-singh 
- For uniformity instead of a setting having three widths values namely full, medium and default for the setting items in expanded condition i defaulted to either full width or reasonable default width <img width="492" alt="Screenshot 2024-03-20 at 8 43 35 AM" src="https://github.com/mattermost/mattermost/assets/17708702/7d7be69d-f0b9-402d-a54c-9890de8bbba5"> 
- For the default width i have chooses margin left as 16% instead of 25% as it gave a bit of breathing room for the settings with lots of options such as (Desktop and mobile notification) and it doesnt look squished

- Previously 
<img width="554" alt="Screenshot 2024-03-20 at 8 48 41 AM" src="https://github.com/mattermost/mattermost/assets/17708702/cb00eb81-3a27-4169-8c16-3260d278cab2">

- In current PR 
<img width="554" alt="Screenshot 2024-03-20 at 8 48 25 AM" src="https://github.com/mattermost/mattermost/assets/17708702/7c915b7a-daef-461f-9328-44c5e61ad31f">

Let me know if you think we need 16% though

Reply:
I like the new 16% margin-left as the default instead of the earlier 25%. We don't need both for sure 👍